api_call_options.go

```diff
@@ -25,7 +26,7 @@ type GetOptions struct {
 	// nested object. Field specifiers can use a dotted notation up to three
 	// references deep. (For example, "memberships.projects" can be used in
 	// ListUsers.)
-	Includes []string `url:"includes,omitempty"`
+	Includes []string `url:"include,omitempty,comma"`
```

Comment:
Good catch. https://pkg.go.dev/github.com/google/go-querystring/query#Values

Reply:
I only noticed after I tested it :)